client/src/components/About/AboutInformation/AboutInformation.js

```diff
@@ -0,0 +1,159 @@
+                Hide
+            </div>
+        );
+
+        return (
+            <div className='about-information-container'>
+                <h1>
+                    About Monuments + Memorials
+                </h1>
+                <p>
+                    <span className='font-italic'>Monuments + Memorials</span> is a crowd-sourced initiative developed at
+                    the Rochester Institute of Technology under the direction of Dr. Juilee Decker. Students in her spring
+                    2019 history course <span className='font-italic'>Monuments & Memory</span> began data collection by
+                    examining 26 states or territories in the US. The following fall, software engineering students in the
+                    Golisano College of Computer and Information Sciences built this web application to house data and to
+                    generate further interest and data collection. To view monuments and memorials, {mapNavLink}. To enter
+                    data, click here.
+                </p>
+                <h3 className='font-italic'>
+                    Background:
+                </h3>
+                <p>
+                    Monuments and memorials offer insight into a community's values. They offer tribute to a person,
+                    idea, or event and/or honor and individual or group who have passed away or have been, in the case
+                    of veterans, killed in action. Across the United States, thousands of such work exists; this vast
+                    collection of publicly-accessible art, however, is largely uncatalogued and thus remains
+                    undocumented for the public. While some familiar outdoor memorials include the Vietnam Veterans
+                    Memorial in Washington, D.C. and the 9/11 Memorial and Museum in NYC -- and such works are deemed
+                    part of our national treasures -- no method exists to enable someone to research and locate public
+                    monuments and memorials throughout the United States. This project aims to remedy that gap in
+                    documentation and access.
+                </p>
+                <p>
+                    Our goal is to map every monument and memorial in the United States and its inhabited territories
+                    (American Samoa, Guam, Puerto Rico, the North Mariana Islands, and U.S. Virgin Islands). We invite
+                    anyone to contribute information and an image of monuments and memorials in these areas. If you have
+                    any questions, please contact the project coordinator, Juilee Decker jdgsh@rit.edu.
+                </p>
+                <h3 className='font-italic'>
+                    Contributors:
+                </h3>
+                <p>
+                    We invite contributions from anyone. If you are interested in contributing, please click here. Your
```

Comment:
This `click here` will also have to be replaced with a link to the sign-in page when that exists.

Reply:
Will do.